docs/reference/release-artifacts.md

```
@@ -43,7 +43,7 @@ We recommend using the TensorRT-LLM NGC container instead of the `ai-dynamo[trtl
 |---------|-------------|--------|----------|------|
 | `ai-dynamo==1.1.0` | Main package with backend integrations (vLLM, SGLang, TRT-LLM) | `3.10`–`3.12` | Linux (glibc `v2.28+`) | [link](https://pypi.org/project/ai-dynamo/1.1.0/) |
 | `ai-dynamo-runtime==1.1.0` | Core Python bindings for Dynamo runtime | `3.10`–`3.12` | Linux (glibc `v2.28+`) | [link](https://pypi.org/project/ai-dynamo-runtime/1.1.0/) |
-| `kvbm==1.1.0` | KV Block Manager for disaggregated KV cache | `3.12` | Linux (glibc `v2.28+`) | [link](https://pypi.org/project/kvbm/1.1.0/) |
+| `kvbm==1.1.0` | KV Block Manager for disaggregated KV cache | `3.10`–`3.12` | Linux (glibc `v2.28+`) | [link](https://pypi.org/project/kvbm/1.1.0/) |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)